### PR TITLE
Fix the issue of COPY failed in Dockerfile

### PR DESCRIPTION
**Pull request by huiwangredhat**
fixes OLCS-243

### CODE REVIEW

**Comment by hanchuntao** [comment]
Here `HOME=/opt/app-root/src`, `-t $OPENLCS_IMAGE:dev -f containers/docker-openlcs/Dockerfile .;`, so the current path also is HOME path, so the change will met same issue.

But this is just my guess, you can remove `only-main` in `build-dev-image` to test whether the change can work.

BTW, `docker-compose.yml` also use it

**Comment by huiwangredhat** [comment]
@hanchuntao, the test of the pipeline is PASS.
https://gitlab.corp.redhat.com/product-security/dev/openlcs/-/pipelines/261744

File: containers/docker-openlcs/Dockerfile [hunk 1]
```diff
@@ -56,7 +56,7 @@ RUN cd /etc/pki/ca-trust/source/anchors/ && \
     chown -R 1001:0 "${HOME}"
 
 # Apply patch for typecode, extractcode, packagedcode
-COPY "${HOME}"/containers/patches/* /opt/app-root/lib64/python3.8/site-packages/
+COPY ./containers/patches/* /opt/app-root/lib64/python3.8/site-packages/
```
[the remark this answers is not in the document]
Comment (huiwangredhat) [reply containers/docker-openlcs/Dockerfile:59]: - Checked line 54 and line 56, and it works.

(app-root) ls -al
total 8
....
drwxrwxrwx. 1 default root  268 Sep  6 03:05 src

- Checked line 55, and it works.
(app-root) ls -al src/containers/docker-openlcs/bin/
total 12
drwxrwxrwx. 1 default root 124 Jul 26 02:40 .
drwxrwxrwx. 1 default root 134 Sep  6 02:34 ..
-rwxrwxrwx. 1 default root 101 Jul 26 02:40 entrypoint-common.sh
-rwxrwxrwx. 1 default root  93 Jul 26 02:40 entrypoint-gunicorn.sh
-rwxrwxrwx. 1 default root 110 Jul 26 02:40 entrypoint-worker.sh

[the remark this answers is not in the document]
Comment (hanchuntao) [reply containers/docker-openlcs/Dockerfile:59]: I still don't understand why ${HOME} cannot work here

If it cannot work, why L54~L56 can work

[the remark this answers is not in the document]
Comment (hanchuntao) [reply containers/docker-openlcs/Dockerfile:59]: OK

**Comment by hanchuntao** [comment]
LGTM

**Pull request merged**